hyperparameter_tuning/rapids_bring_your_own/rapids_sagemaker_hpo.ipynb

```diff
@@ -833,7 +876,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "estimator = sagemaker.estimator.Estimator( **estimator_params  )"
+    "estimator = sagemaker.estimator.Estimator(**estimator_params)"
```

Comment:
This won't work with v2 which is rolling out everywhere right now.
Seeing this:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-36-aee0325cbb5f> in <module>
----> 1 estimator = sagemaker.estimator.Estimator(**estimator_params)

TypeError: __init__() missing 1 required positional argument: 'image_uri'
```

Reply:
Recommend updating your kernel to v2.15 or above and retesting...